src/emoji/EmojiRow.js

```diff
@@ -39,21 +37,17 @@ class EmojiRow extends PureComponent<Props> {
   };
 
   render() {
-    const { name, imageEmoji } = this.props;
+    const { name } = this.props;
 
-    // TODO: this only handles Unicode emoji (shipped with the app)
-    // and realm emoji, but not Zulip extra emoji.  See our issue #2846.
     return (
       <Touchable onPress={this.handlePress}>
         <View style={styles.emojiRow}>
-          {imageEmoji ? <ImageEmoji emoji={imageEmoji} /> : <Emoji name={name} size={20} />}
+          <Emoji name={name} />
           <RawLabel style={styles.text} text={name} />
         </View>
       </Touchable>
     );
   }
 }
 
-export default connect((state, props): SelectorProps => ({
-  imageEmoji: getActiveImageEmojiByName(state)[props.name],
-}))(EmojiRow);
```

Comment:
Please look again at this and clean up everything that's no longer being used here.